multihost_test/bz_automation/test_automation.py

```diff
@@ -127,3 +127,27 @@ def test_2091062(self, multihost, create_localusers):
         client.logout()
         assert "pam_motd: error scanning directory" not in \
                execute_cmd(multihost, "cat /var/log/secure").stdout_text
+
+    def test_sssd_6788(self, multihost, create_localusers, bkp_pam_config):
```

Comment:
Change the function name to something more meaningful like "test_bcrypt_support"